tests/test_geometry.py

```diff
@@ -90,7 +90,7 @@ def test_assign_truncated_sildenafil():
         LG.G4_TETRAHEDRAL,  # S
         LG.G1_TERMINAL,  # O-S
         LG.G1_TERMINAL,  # O-S
-        LG.G3_PYRAMIDAL,  # N
+        LG.G3_PLANAR,  # N, note, rdkit hybridization thinks this is pyramidal
```

Comment:
So there is an improper which makes it planar? That seems surprising to me.

Reply:
Yes - the improper is probably this one:

```
['[*:1]~[#7X3$(*~[#6X3]):2](~[*:3])~[*:4]', 1.3946666666666667, 3.141592653589793, 2.0],
```

Reply:
Note that presence of an improper doesn't guarantee that it's planar - it depends on the strength of the force constants etc relative to other forces that may be at play. But the intention of impropers is to planarize.